app/ui/src/HttpArchive.elm

```diff
@@ -0,0 +1,145 @@
+module HttpArchive exposing (..)
+
+import Json.Decode exposing (..)
+import Date
+
+
+type alias Header =
+    { name : String
+    , value : String
+    }
+
+
+type alias Request =
+    { method : String
+    , url : String
+    , headers : List Header
+    , headersSize : Int
+    , bodySize : Int
+    }
+
+
+type alias Content =
+    { size : Int
+    , mimeType : String
+    , text : Maybe String
+    }
+
+
+type alias Response =
+    { status : Int
+    , statusText : String
```

Comment:
Not yet, we just display it as text. Defining an enum for that seems like an overkill...